src/pages/Compare/index.tsx

```diff
@@ -159,7 +194,8 @@ export class CompareBase extends React.Component<Props> {
         </Helmet>
         <VersionFileViewer
           compareInfo={compareInfo}
-          file={null}
+          file={compareInfo && compareInfo.diff ? versionFile : null}
```

Comment:
Is there a test case for when `compareInfo` is not defined? I only see a test case for when `compareInfo.diff` is `null` in this patch.

Reply:
13 tests fail if I remove `compareInfo &&` but, more importantly, there is a type error which points directly to the problem. I think it's fine to rely on the type checking for this.